serverless/serverless
# Add validation to remove command

## What did you implement:

Closes #4406.

Serverless now throws an error if `sls remove` is run in a directory without a `serverless.yml`, rather than exiting silently.

## How did you implement it:

Added `validate` to the `before:remove:remove` hook. This mirrors the implementation in the [deploy command](https://github.com/serverless/serverless/blob/master/lib/plugins/deploy/deploy.js).

## How can we verify it:

1. Pull this branch down.
2. Change into a directory without a `serverless.yml` file.
3. Run `<path-to-repo>/bin/serverless remove`. It should throw an error.

## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
